src/services/firebase-client/rtdb/index.ts

```diff
@@ -0,0 +1,49 @@
+import firebase from "firebase/app"
+import "firebase/database"
+import { configs as firebaseConfigs } from '../../../constants/firebase'
+
+const isFirebaseInitialized = (): boolean => !!firebase.apps.length
+
+if (!isFirebaseInitialized()) {
+  firebase.initializeApp(firebaseConfigs)
+}
+
+const db = firebase.database()
```

Comment:
this module looks like a wrapper caller on firebase's realtime database
This could be great if you plan to switch database in the future but in my opinion, its unnecessary.
It's better to have an abstraction layer on business logic instead of database implementation.